README.md

```diff
@@ -504,7 +505,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Crislana Rafael](https://github.com/crislanarafael)
   - [Cux Sabrina](https://github.com/theflucs)
   - [Cynthia Teeters](https://github.com/cynthiateeters)
-  - [Chijioke Egbo](https://github.com/chijiokeegbo)
+
```

Comment:
Hey @chijiokeegbo , Could you remove this extra space so that we can merge your Pull Request.